skimage/morphology/tests/test_misc.py

```diff
@@ -163,7 +164,8 @@ def test_uint_image_holes():
                          [0, 0, 0, 0, 0, 0, 0, 1, 1, 1],
                          [0, 0, 0, 0, 0, 0, 0, 1, 1, 1],
                          [0, 0, 0, 0, 0, 0, 0, 1, 1, 1]], dtype=np.bool_)
-    observed = remove_small_holes(labeled_holes_image, area_threshold=3)
+    with expected_warnings(['Any labeled images will be returned as a boolean array.']):  # noqa
```

Comment:
You don't need the full string here.

Reply:
Was still learning to use `expected_warnings`.

Reply:
Sure! But please update so that it's PEP8 friendly and the noqa is unnecessary. =)

Reply:
I guess I had the same line in two locations. Thanks for spotting this one.